server/tests/src/test/java/org/infinispan/server/persistence/PersistenceIT.java

```diff
@@ -21,16 +39,64 @@
 })
 public class PersistenceIT {
 
-   public static final DatabaseServerListener DATABASE = new DatabaseServerListener("h2", "mysql", "postgres");
+   private static final String DATABASES = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_TYPES);
+   private static final String DATABASE_LIBS = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_LIBS);
+   private static final Integer ARTIFACT_ID_POSITION = 1;
+   private static final String EXTERNAL_JDBC_DRIVER = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_EXTERNAL_DRIVERS);
+   private static String jdbcDriverFromFile = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_DRIVERS_FILE);
```

Comment:
This can be:

```java
private static final String JDBC_DRIVER_FROM_FILE = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_CONTAINER_DATABASE_DRIVERS_FILE, "src/test/resources/database/jdbc-drivers.txt");
```